code/script.js

```diff
@@ -1,19 +1,198 @@
-// Start here
 
 // Step 1 - Welcome and introduction
-// Your code goes here
 alert(
-  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
+  `Hey there, pizza lover! 🍕 Welcome to Pizzeria Fresco! Click 'OK' to get started!`
+)
+const userName = prompt(`What is your name?`)
+console.log("User's name is:", userName)
+
+alert(
+  `Hi there, ${userName}! 😊 It's great to meet you. Click 'OK' to start placing your order.`
 )
 
 // Step 2 - Food choice
-// Your code goes here
+// Selection between Pizza, Pasta, Salad
+// while loop to ensure that the user selects a valid option if not, the user will be prompted to select again. Will loop until it hits break. 
+let foodType
+
+while (true) {
+
+  let foodSelection = prompt(`Please select your food of choice: 
+      Write: 1 - for Pizza 🍕
+      Write: 2 - for Pasta 🍝
+      Write: 3 - for Salad 🥗`)
+
+  // convert the variabel "foodSelection" to be a number instead of a string
```

Comment:
Good job converting the user input to a number. That made your conditional statement cleaner, and ensures that the comparison works as expected.